dradis/dradis-ce
# Notifications creation

### Spec
We comment on a model, let's say an issue, that we didn't create.
Notifications must be stored on db, so the user that created that issue and the rest of subscribers may see it in their notifications list.

No notification should be created for the comment creator.

We may trigger the notifications creation when activity is created. But it must be done in the background.

**Proposed solution**
We may schedule bg job from the ActivityTracking concern, that creates the necessary notifications.


### How to test
If https://github.com/dradis/dradis-ce/pull/306 is merged:
  - Create a comment on an issue
  - Notifications should be created in db for all users subscribed to that issue except for the commenter

Else:
  - Specs should passs
